### DISCUSSION TITLE
Allow creating signed urls from FileService

### DISCUSSION THREAD

**Discussion by erickjth** [directus/directus#14884] [Archive]
Hi there,

Sometimes there is a necessity to expose some files from the CMS storage to the outside in custom endpoints via extensions. 

I realized that in almost all drivers there is a `getSignedUrl` method to create this kind of link, so I am wondering why not have something accessible from the FileService to access this method. Probably for the local driver, should be no support.

Thanks.

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/7612 🙂
